code/__DEFINES/atmospherics/turfs.dm

```diff
@@ -0,0 +1,3 @@
+// /turf/var/air_flags flags
+/// If this is there, the turf will apply graphics even if air status isn't AIR_STATUS_NORMAL. Otherwise, the turf will have no gas graphics, ever.
```

Comment:
I dont understand this comment:
Graphics are only applied when: 1. Air status isnt normal or 2. this flag is set.
But the second sentence says there is no graphics ever?

Reply:
i just deleted it it's a sneak peek of a new thing i'm doing later regarding unsimulated